config/SGGame/pokemon.json

```diff
@@ -17426,12 +17426,40 @@
 	},
 	"necrozmaduskmane": {
 		"id": "necrozmaduskmane",
-		"inherit": "necrozma"
+		"inherit": "necrozma",
+    "evDrops": {
+			"hp": 0,
+			"atk": 0,
+			"def": 0,
+			"spa": 3,
+			"spd": 0,
+			"spe": 0
+		}
```

Comment:
Specify totalEvDrops even though it matches necrozma, incase necrozmas changes so it dosent match its other formes anymore.

Reply:
Ok